### ISSUE TITLE
Babel transform error in victory-core 9.1.0

### ISSUE THREAD

**Issue by rgoldiez** [FormidableLabs/victory#416]
Getting a babel error on a fresh install of `victory-native`.  Error looks to be in `victory-core`

```
TransformError: /Users/rgoldiez/Documents/hirebotics/mobile/hb-react-app/app/node_modules/victory-chart/src/index.js: [BABEL] /Users/rgoldiez/Documents/hirebotics/mobile/hb-react-app/app/node_modules/victory-chart/src/index.js: Couldn't resolve extends clause of builder-victory-component/config/babel/.babelrc in /Users/rgoldiez/Documents/hirebotics/mobile/hb-react-app/app/node_modules/victory-chart/.babelrc
```

**Comment by ryan-roemer** [comment]
@boygirl - this situation is likely going to need something like '.npmignore.publishr' that excludes babelrc (can't be in normal one or else git installs won't work)

As you mentioned this in https://github.com/FormidableLabs/victory-docs/pull/175 it may be part of that solution as well if there we don't actually build off 'src' but rather only 'lib'

**Comment by hybrisCole** [comment]
I'm getting this exact same thing, do you guys know a workaround to make this error disappear?

Regards,
![image](https://cloud.githubusercontent.com/assets/951133/19864642/b4f009f2-9f5e-11e6-854a-24da0ff86a38.png)

**Comment by boygirl** [comment]
@hybrisCole This is in flight, and should be released as a patch today.

**Comment by boygirl** [comment]
@rgoldiez @hybrisCole This fix has just been released as `victory@0.13.3`
